File: src/main/java/com/marknkamau/Exceptions/EmptyInputFile.java
package com.marknkamau.Exceptions;

public class EmptyInputFile extends BaseException {
    private static final String TITLE = "Empty input file";
    private static final String EMPTY_INPUT_FILE = "Input file is empty.";

    public EmptyInputFile() {
        super(TITLE, EMPTY_INPUT_FILE);
    }
}

File: src/main/java/com/marknkamau/Exceptions/EmptyInputFile.kt
package com.marknkamau.Exceptions

class EmptyInputFile : BaseException(EmptyInputFile.TITLE, EmptyInputFile.EMPTY_INPUT_FILE) {
    companion object {
        private val TITLE = "Empty input file"
        private val EMPTY_INPUT_FILE = "Input file is empty."
    }
}
